README.md

```diff
@@ -1,29 +1,79 @@
-# Customer Service Conversational Insights with Azure OpenAI
+# Conversation Knowledge Mining
 
-## Knowledge Mining applied to Conversations
+MENU: [**OVERVIEW**](#use-case-overview) \| [**GETTING STARTED**](#getting-started) \| [**EXTENSIBILITY**](https://github.com/rturknett/Customer-Service-Conversational-Insights-with-Azure-OpenAI-Services/blob/readMe-updates/Extensibility.md) \| [**AUDIO DATA**](#audio-data) \|
+[**HOW TO USE**](#how-to-use) \| [**TROUBLESHOOTING**](https://github.com/rturknett/Customer-Service-Conversational-Insights-with-Azure-OpenAI-Services/blob/readMe-updates/Troubleshooting.md)
 
-Every day a large amount of data is generated by Customer Services in the form of Natural Language conversations with customers. This data is stored as chat logs (with human agents or bots) or audio recordings from voice-based calls.
+## **USE CASE OVERVIEW:**
```

Comment:
```suggestion
## USE CASE OVERVIEW
```
Headings should not contain colons or boldface (same for all other headings in the documentation)

Reply:
The all-caps also looks inconsistent with other headings, and it isn't clear what it is trying to convey